dev/Deployment/DeploymentManager.cpp

```diff
@@ -298,20 +300,19 @@ namespace winrt::Microsoft::Windows::ApplicationModel::WindowsAppRuntime::implem
         wil::unique_hmodule module;
         THROW_IF_WIN32_BOOL_FALSE(GetModuleHandleEx(GET_MODULE_HANDLE_EX_FLAG_FROM_ADDRESS, reinterpret_cast<PCWSTR>(DeploymentManager::GenerateDeploymentAgentPath), &module));
 
-        std::filesystem::path modulePath = wil::GetModuleFileNameW<std::wstring>(module.get());
+        std::filesystem::path modulePath{ wil::GetModuleFileNameW<std::wstring>(module.get()) };
         return modulePath.parent_path() / c_deploymentAgentFilename;
     }
 
     HRESULT DeploymentManager::AddPackageInBreakAwayProcess(const std::filesystem::path& packagePath, const bool forceDeployment) try 
     {
-        auto exePath = GenerateDeploymentAgentPath();
-        WCHAR szActivityId[64]{0};
-        StringFromGUID2(*::WindowsAppRuntime::Deployment::Activity::Context::Get().GetActivity().Id(), szActivityId, 64);
+        auto exePath{ GenerateDeploymentAgentPath() };
+        auto szActivityId{ winrt::to_hstring(*::WindowsAppRuntime::Deployment::Activity::Context::Get().GetActivity().Id()) };
```

Comment:
`activityId`

`sz` prefix is Hungarian Notation which is soooooo last century ;-)

It's also expressly discouraged per the coding standards